### ISSUE TITLE
“All the links” experiment

### ISSUE THREAD

**Issue by simevidas** [simevidas/webplatformdaily-site#105]
I’m experimenting with posting even more links at the bottom of daily releases on http://webplatformdaily.org. Here’s the gist:

 - foo

**Comment by aaronbuchanan** [comment]
:thumbsup

**Comment by sarukuku** [comment]
:+1:

**Comment by teodragovic** [comment]
Suggestion: it's hard to quickly scan additional links when they are in paragraph. Maybe use bullet points like with main content but with smaller font size and toggle/show more switch?

**Comment by sarukuku** [comment]
I agree with @teodragovic. Currently it's hard to scan.

**Comment by simevidas** [comment]
@teodragovic @sarukuku yeah, I'll do bullet points; I plan to properly integrate this feature with my build tasks and style sheets over the weekend ^_^
